tests/test_teststats.py

```diff
@@ -16,7 +16,7 @@ def test_qmu(caplog):
         pyhf.infer.test_statistics.qmu(
             mu, data, model, init_pars, par_bounds, fixed_params
         )
-        assert "WARNING  qmu test statistic used for fit" in caplog.text
+        assert "qmu test statistic used for fit" in caplog.text
```

Comment:
Can you elaborate on why do these warnings need to get removed?

Reply:
Two reasons:

1) you’re already using caplog to specify warning log level so no need to check that.
2) pytest changes how logging is handled and there are most spaces and extra text between the warning and the actual message... the test should only care about the message, not depending on how the logger is formatted.